### DISCUSSION TITLE
Priority of applications in Django

### DISCUSSION THREAD

**Discussion by estuardodev** [reactive-python/reactpy-django#158] [Problem] [answered]
There is a problem regarding what are the applications that need to be installed.

Since the documentation indicates that we must do the following:
1. `pip install reactpy-django`
2.
 ```
INSTALLED_APPS = [
    "reactpy_django",
    ...,
]
```

But the problem that points out that it should be at the beginning and what happens is that if you put at the beginning **"reactpy_django"** then the django manager does not work and throws error.

It should be placed in this way so that no errors occur:
```
INSTALLED_APPS = [
    'daphne', # Required up to the top

    'django.contrib.admin', # Django Apps
    'django.contrib.auth', # Django Apps
    'django.contrib.contenttypes', # Apps by Django
    'django.contrib.sessions', # Apps by Django
    'django.contrib.messages', # Django Apps
    'django.contrib.staticfiles', # Apps by Django

    'reactpy_django', # Here mandatory
]
```

I think this problem should be solved in the documentation and clarified correctly for those new to this.

**Comment by Archmonger** [comment, marked as answer]
Yup you're right.

The ellipsis (`...`) in `INSTALLED_APPS` should definitely come before `reactpy_django`.

Do you want to PR this?

**Reply by estuardodev** [reply]
Correct. I would really like to do the PR. Thank you.
